src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -138,9 +138,10 @@ type ClusterStatus struct {
 
 // NodesList shows where client can find Redpanda brokers
 type NodesList struct {
-	Internal      []string `json:"internal,omitempty"`
-	External      []string `json:"external,omitempty"`
-	ExternalAdmin []string `json:"externalAdmin,omitempty"`
+	Internal           []string `json:"internal,omitempty"`
+	External           []string `json:"external,omitempty"`
+	ExternalAdmin      []string `json:"externalAdmin,omitempty"`
+	ExternalPandaproxy []string `json:"externalPandaproxy,omitempty"`
```

Comment:
should we call it just ExternalProxy?

Reply:
I guess if we go that way, it should be in all places. I just find it very long and confusing that it's one word :D

Reply:
It's "pandaproxy" in the docs and (cpp) source code, so I thought let's follow that..